docs/reference/cloud/elastic-cloud-kibana-settings.md

```diff
@@ -74,6 +74,18 @@ stack: ga 9.1
 `xpack.banners.linkColor`
 :   The color for the banner link text. Defaults to `#0B64DD`.
 
+`xpack.product_intercept.enabled`
+:   Enable or disable Elastic product feedback prompts. Defaults to `true`.
+
+`xpack.product_intercept.interval`:
+:   Set the time that elapses between Elastic product feedback prompts. The time is formatted as a number and a time unit (d,h,m,s). For example, 20m, 24h, 7d. Defaults to `90d`.
```

Comment:
these appeared during conflict resolution, I double checked and these settings were made available to cloud in https://github.com/elastic/cloud/pull/141517 and does apply to 9.1 so should be fine to backport here